hooks/Logout/useLogout.tsx

```diff
@@ -0,0 +1,20 @@
+import { useAuthContext } from "@context/Authentication"
+import { AuthDispatchTypes } from "@context/Authentication/AuthDispatchTypes";
+import { useRouter } from "next/router";
+
+const useLogout = () => {
```

Comment:
Hello Alex, if I may ask, where is this hook being used? I can't seem to find it :D

Reply:
Hello, rashad. I used the useLogout hook on one of the dropdowns in the navbar. The hook returns the logout function that dispatches a logout event to the reducer. Hope it helps! 😁

Reply:
Oh okay got it Alex, thanks!